src/styles/index.scss

```diff
@@ -1,3 +1,106 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16;
```

Comment:
The `margin-bottom` property is missing a unit. It should be `16px` instead of just `16` to follow CSS standards.